README.md

```diff
@@ -92,6 +92,27 @@ Pylance provides users with the ability to customize their Python language suppo
         - `true`
         - `false`
 
+-   `python.analysis.autoImportCompletions`
```

Comment:
`python.analysis.autoImportsEnabled` ?

Reply:
option name is pre-existing one. I don't think we want to change it. it is not something I just added.

Reply:
I just moved the option up in the README file so that auto import related options are listed together.